engines/scumm/imuse_digi/dimuse_music.cpp

```diff
@@ -299,7 +321,7 @@ void IMuseDigital::playComiMusic(const char *songName, const imuseComiTable *tab
 		fadeOutMusic(120);
 		return;
 	}
-
+	
```

Comment:
formatting

Reply:
I'm sorry, I don't understand, that function is unchanged from the previous pull request.